travis_test.m

```diff
@@ -5,9 +5,10 @@
 [s, v] = bfCheckJavaPath()
 bfInitLogging('INFO');
 
-test_bfopen('tubhiswt-3D/tubhiswt_C0.ome.tif');
+arg_list = argv();
+test_bfopen(arg_list{1});
 
 % Increase the debugging verbosity
 javaMethod('setRootLevel', 'loci.common.DebugTools', 'DEBUG');
 
-test_bfopen('tubhiswt-3D/tubhiswt_C0.ome.tif');
+test_bfopen(arg_list{2});
```

Comment:
Comment from call: I find the hard-coded `2` a bit odd. I'd either run multiple dockers passing `arg_list{1}` to each, or loop over all of `arg_list`